Assignment3/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace CanHazFunny;
+
+public class Jester
+{
+
+    public IJokeService JokeService { get; }
+    public IOutputJoke OutputJoke { get; }
```

Comment:
```suggestion
    private IJokeService _JokeService;
    private IOutputJoke _OutputJoke;
```
If these don't need to be public, make them private.